ember-server/shared/src/test/scala/org/http4s/ember/server/EmberServerSuite.scala

```diff
@@ -193,4 +196,19 @@ class EmberServerSuite extends Http4sSuite {
     }
   }
 
+  fixture().test(
+    "#7655 — client shouldn't fail request processing when server ignores the request body"
+  ) { case (server, client) =>
+    def runReq[A: EntityEncoder[IO, *]](server: Server, entity: A) = {
+      val req =
+        Request[IO](Method.POST, uri = url(server.addressIp4s, "/ignored-body/what"))
+          .withEntity(entity)
```

Comment:
It looks like this test design is using a non-chunked body. My demo code had a lazy body, which caused the Ember client to send 2 packets. I think this is why you need a much larger body to reproduce.